src/content/en/2020/http2.md

```diff
@@ -4,20 +4,534 @@ part_number: IV
+
+There is ongoing work to define ways to discover HTTP/3 without needing the TCP step. This should be considered an optimization though, as the UDP blocking issues are likely to mean that TCP-based HTTP sticks around. The [HTTPS DNS record](https://tools.ietf.org/html/draft-ietf-dnsop-svcb-https), is similar to HTTP Alternative Services and some CDNs are already [experimenting with these records](https://blog.cloudflare.com/speeding-up-https-and-http-3-negotiation-with-dns/).  In the long run, when most servers offer HTTP/3, browsers might switch to attempting that by default. But that will take a long time.
+
+QUIC is dependent on TLS1.3, which is used for around 41% of requests, which means the other 60% of requests will need to update their TLS stack to support HTTP/3.
+
+#### Dual stack deployment implications
+
+From the discussion above, we expect QUIC and TCP to continue running in parallel for some time. We expect servers (e.g., nginx/apache/nodejs) to offer easy ways to do so, but this is still likely to complicate localhost test setups. Furthermore, there's probably a longer term impact on the HTTP Archive and WebPageTest projects.  The amount of pages crawled will increase substantially if we want to crawl both the HTTP/3 and H2/H1 versions, or the runtimes will increase if we want to prioritize HTTP/3.
+
+Currently, some browsers provide (temporary) command line arguments to force QUIC on a certain origin to make this easier, but this is not the most user-friendly option of course. Over time, this will probably become easier to configure. For example, this command will force Chrome to try a QUIC connection to localhost immediately without having to discover it via alternative services :
+
+```
+chrome --enable-quic --quic-version=h3-29 --origin-to-force-quic-on=localhost:6121 http://localhost:6121/
+```
+
+### Not everything is implemented
+
+So when can we start using HTTP/3 and QUIC for real? Not quite yet, but hopefully soon. There's a [large number of mature open source implementations](https://github.com/quicwg/base-drafts/wiki/Implementations) and the major browsers have experimental support. However, most of the typical servers have suffered some delays: nginx is a bit behind other stacks, apache hasn’t announced official support, and NodeJS relies on OpenSSL, which [won’t add QUIC support anytime soon](https://github.com/openssl/openssl/pull/8797). Next to this, some features like connection migration, HTTP/3's new prioritization scheme, or 0-RTT support aren’t widespread yet.
+
+As such, you yourself can start experimenting with HTTP/3 today, but be careful with drawing too general conclusions from the performance that you’re seeing today!
+
+## Conclusion
+
+{# TODO This Conclusion seems like a conclusion to HTTP/3 rather than to the whole chapter. Should we add a paragraph concluding HTTP/2's usage on top of this? #}
```

Comment:
Yes. :D

I suggest renaming `### Not everything is implemented` to `### Looking ahead`, then combine/deduplicate the following text with the text already in that section. Then, `## Conclusion` can pick out some of the most interesting trends presented for HTTP/2 and simply say "HTTP/3 is around the corner, we expect next year's Almanac to see some interesting new data"

Reply:
See what you think now.

Reply:
LGTM!

Slight niggle would be "Looking Toward the Future: HTTP/3" and "Looking ahead" are very similar. Maybe change latter to "Is HTTP/3 ready for use yet?", or "Is HTTP/3 available now?" Or similar?

Anyway shall we merge this and revisit in editing?

Reply:
updated .. ready to merge